proposals/0221-multiple-modules.md

```diff
@@ -570,15 +576,15 @@ It is not scalable for many parameters.
 
 #### Option 2: OEM assigned ID
 
-In this option, instead of saying a module is located in a grid, an OEM assigns an ID to each module. The ID may be a UUID that does not have any meaning that can be understood by a user or an app. The ID may be a meaningful ID for the module, for example, a seat number assigned by the OEM or anything meaningful that can allow a normal user/driver/passenger to associate the ID to the module with just common sense.
+In this option, instead of saying a module is located in a grid, an OEM assigns an ID to each module （Basically, only `moduleId` in `ModuleInfo` is used, all optional parameters are not used). The ID may be a UUID that does not have any meaning that can be understood by a user or an app. The ID may be a meaningful ID for the module, for example, a seat number assigned by the OEM or anything meaningful that can allow a normal user/driver/passenger to associate the ID to the module with just common sense.
```

Comment:
```suggestion
In this option, instead of saying a module is located in a grid, an OEM assigns an ID to each module（basically, only `moduleId` in `ModuleInfo` is used, all optional parameters are not used). The ID may be a UUID that does not have any meaning that can be understood by a user or an app. The ID may be a meaningful ID for the module, for example, a seat number assigned by the OEM or anything meaningful that can allow a normal user/driver/passenger to associate the ID to the module with just common sense.
```

Reply:
changed